### PR TITLE
Writing pull_request.md manual page

### PR DESCRIPTION
**Pull request by trebmuh**
As suggested in #28.

### CODE REVIEW

File: _manual/pull_request.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ manual: cont
 order: 30
 ---
 
-_...to be written..._
+## Some rules
```
Comment (peterdd) [inline _manual/pull_request.md:8]: 'rules' sound too harsh IMHO.

File: _manual/pull_request.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ manual: cont
...
 
-Isn't a good way for you to **start [contributing documentation](/manual/writing_documentation)?**
+- Please do not put too much into one PR. To ease reviewing and testing, it is prefered that a pull request handles only one single problem. That because one obstacle/problem and it delays the possible good stuff to merge.
+- Please give pull requests a good summary, `$name fixes` as an example, to make the review process easier.
```
Comment (peterdd) [inline _manual/pull_request.md:11]: Oh, sorry, you misunderstood. That was a bit ironic reference for your 'Trebmuh fixes' PR title..

File: _manual/pull_request.md [hunk 3]
```diff
@@ -5,6 +5,10 @@ manual: cont
...
+- Note that the correct spelling is `Flyspray` (just a **F** capital letter).
```
Comment (peterdd) [inline _manual/pull_request.md]: I do not think this should be here.

File: _manual/pull_request.md [hunk 4]
```diff
@@ -5,6 +5,10 @@ manual: cont
...
+
+If you whish to, you can **[contribute documentation](/manual/writing_documentation)** here.
```
Comment (peterdd) [inline _manual/pull_request.md]: whish -> wish
The 'to'  is right here? My english grammar is bad...

**Review by peterdd — COMMENTED** [review]
made some comments to your PR
(it is my first new Github 'Review' workflow)

File: _manual/pull_request.md (hunk 1 above)
Comment (trebmuh) [reply _manual/pull_request.md:8]: advices ?

File: _manual/pull_request.md (hunk 2 above)
Comment (trebmuh) [reply _manual/pull_request.md:11]: Still a good advice, isn't?

File: _manual/pull_request.md (hunk 3 above)
Comment (trebmuh) [reply _manual/pull_request.md]: OK, removed.

File: _manual/pull_request.md (hunk 4 above)
Comment (trebmuh) [reply _manual/pull_request.md]: whish/wish fixed.

Yes, the "to" is correct.

File: _manual/pull_request.md (hunk 3 above)
Comment (trebmuh) [reply _manual/pull_request.md]: Removed from this `pull_request.md` page, and re-added in `writing_documentation.md`.

File: _manual/pull_request.md (hunk 1 above)
Comment (peterdd) [reply _manual/pull_request.md:8]: Suggestions? I do not know, just do not want people feel to restricted when not absolute necessary.

Comment (trebmuh) [reply _manual/pull_request.md:8]: Agreed with "suggestions". Done in #30.

**Pull request merged**